src/renderer/components/settings-page/index.tsx

```diff
@@ -92,7 +91,7 @@ export const SettingsPage: FC<{
           </Button>
         </Row>
       </Container>
-      <NextButton text={t('Confirm')} onClick={onNext} disabled={false} />
+      <NextButton text={t('OK')} onClick={onSelect} disabled={false} />
```

Comment:
Is this a new string? If so, you should add it to the locales file for translation.

Reply:
Added in Chinese and English locales file